Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM alpine:3.18
 
+RUN apk --no-cache add curl
```

Comment:
`curl` is required to run the K8S healthcheck to make an HTTP Post call to the Buf connect server.
```yaml
livenessProbe:
  exec:
    command:
      - curl
      - -X
      - POST
      - -H
      - "Content-Type: application/json"
      - -d
      - '{"service":"raystack.frontier.v1beta1.FrontierService"}'
      - http://localhost:8002/grpc.health.v1.Health/Check
  initialDelaySeconds: 15
  periodSeconds: 10
```